src/app/faq-block/faq-block.component.ts

```diff
@@ -0,0 +1,31 @@
+import {Component, Input, OnInit} from '@angular/core';
+
+interface Faq {
+  question: string;
+  answer: string;
+}
+
+@Component({
+  selector: 'app-faq-block',
+  templateUrl: './faq-block.component.html',
+  styleUrls: ['./faq-block.component.scss']
+})
+export class FaqBlockComponent implements OnInit {
+
+  @Input() headline: string;
+  @Input() json: string;
+
+  toggle = [];
+
+  faq: Faq[];
+
+  constructor() {
+  }
+
+  ngOnInit(): void {
+    import('src/assets/data/' + this.json).then(data => {
```

Comment:
Maybe you can once more check if you can use the HttpClient approach instead of `import` to load the json. This would make it easier to switch to statically served files (e.g. through S3) later down the road.

Reply:
In order to load the json correctly from the asset folder, you will most likely only have to remove the `src` from the path.

Reply:
Done